code/script.js

```diff
@@ -33,19 +48,135 @@ const showMessage = (message, sender) => {
+const showYesNo = (name) => {
+  showMessage(`${name}? That's a lame name, dude. Did your mommy and daddy choose that for you?`, 'bot');
+  
+  inputWrapper.innerHTML = `
+    <button id="yesBtn" type='button'>Yes</button>
+    <button id="noBtn" type='button'>No</button>`
+    
+  document
+    .getElementById('yesBtn').addEventListener('click', Yes)
+  document
+    .getElementById('noBtn').addEventListener('click', No)
+    
+}
+       
+  const Yes = () => {
+    showMessage("Yes, actually.", 'user')
+    showMessage("Don't care, didn't ask", 'bot')
+  
+    setTimeout(lunchMoney, 500)
+}
+
+  const No = () => {
+    showMessage("Nope! I chose it for myself.", 'user')
+    showMessage("What a weirdo!", 'bot')
+    setTimeout(lunchMoney, 500)
+}
+
+
+
+// Third interaction between bot and user
+const lunchMoney = (event) => {
+  showMessage("Give me your lunch money, nerd!", 'bot')
+  
+  inputWrapper.innerHTML = `
+    <button id="giveMoney" type='button'>Okay fine!</button>
+    <button id="keepMoney" type='button'>Hell no!</button>`
+    
+  document
+    .getElementById('giveMoney').addEventListener('click', giveMoney)
+  document
+    .getElementById('keepMoney').addEventListener('click', keepMoney)
+    
+
+    event.preventDefault()
+  }
+
+  const giveMoney = () => {
+    showMessage("Okay fine! Just please don't hurt me!", 'user')
+    setTimeout(showDropdown, 500)
+  }
+  const keepMoney = () => {
+    showMessage("Hell no! You're just a BULLY!", 'user')
+    setTimeout(showDropdown, 500)
+  }
+
+  const showDropdown = () => {
+    showMessage("What are you gonna do now, loser?", 'bot');
+    inputWrapper.innerHTML = `
+    <select id="select">
+          <option value="" selected="" disabled="">Select your fate...</option>
+          <option value="Cry">*Start crying*</option>
+          <option value="Tattle">*Run and tell someone*</option>
+          <option value="Fight">*Raise your fist in retaliation*</option>
+    </select>`
+
+      document.getElementById('select').addEventListener('change', Ending)
+     
+    } 
+
+const Ending = (event) =>{
+  showMessage(event.target.value,'user');
+  if (event.target.value === "Cry") {
+  setTimeout(() =>showMessage("Oh my god, shut up! You're so annoying", 'bot'),1000);
+}
+
+else if (event.target.value === "Tattle") {
+  setTimeout(() =>showMessage("You just made a huge mistake!", 'bot'),1000);
+ 
+}
+
+else {
+  setTimeout(() =>showMessage("Ha, you're joking right? You're so dead.", 'bot'),1000);
+}
+ 
+  setTimeout(() =>
+  inputWrapper.innerHTML = `
+  <label for ="name-input">You have been knocked out by BullyBot</label>`,1000);
+
+  let punch = new Audio('assets/umph-47201.mp3')
+  punch.play();
+
+}
+
+
+
 // Set up your eventlisteners here
 
+
+inputWrapper.addEventListener('submit', handleNameInput);
+inputWrapper.addEventListener('button', showYesNo)
+inputWrapper.addEventListener('button', lunchMoney)
+
 // When website loaded, chatbot asks first question.
```

Comment:
For even more clarity I think you could maybe place these comments in the place where it happens in the code.